shell/platform/darwin/macos/framework/Source/FlutterAppLifecycleDelegate.mm

```diff
@@ -87,7 +82,7 @@ - (void)addDelegate:(NSObject<FlutterAppLifecycleDelegate>*)delegate {
 
 - (void)removeDelegate:(NSObject<FlutterAppLifecycleDelegate>*)delegate {
   NSUInteger index = [[_delegates allObjects] indexOfObject:delegate];
-  if (index >= 0) {
+  if (index != NSNotFound) {
```

Comment:
Noticed while I was looking at the existing structure; the previous code was wrong since `index` is unsigned. `NSNotFound` is the unsigned version of -1, but not actually negative.

Reply:
Nice catch.